app/locales/de.json

```
@@ -1,20 +1,36 @@
 {
-  "title": "Innovationserhebung",
   "disclaimer": "Daten aus 10 Jahre Innovationserhebung ab Innovationserhebung 2013",
+  "hero": {
+    "imageAlt": "Logo der Technologiestiftung Berlin",
+    "title": "Innovationsmonitor der Stadt Berlin",
+    "subTitle": "Innovationsverhalten der Berliner Wirtschaft von 2012 – 2023"
+  },
+  "nav": {
+    "alignIconAlt": "Inhaltsverzeichnis",
```

You might want to consider renaming the key here to avoid confusion, e.g.:

```suggestion
    "tableOfContents": "Inhaltsverzeichnis",
```